mobify/progressive-web-scaffold
# WEB-1112: Design feedback fixes

Address WEB-1099 design feedback for Payment Step (WEB-1112)

 **JIRA**:
https://mobify.atlassian.net/browse/WEB-1112
https://mobify.atlassian.net/browse/WEB-1099

## Changes
- Adjust various spacing as per design feedback
  - Tweaked Accordion's padding and units to match latest checkout mocks (44px height instead of 48px)
  - Tweaked Ledger's padding
- Use `CardInput` component from the SDK to get auto number formatting
- Removed `u-bg-color-neutral-10` from checkout payment container into `app.scss` modifier styles
   - Previously, the background color wasn't stretching to cover the available space leaving undesired whitespace between the main content and the footer. (Can easily reproduce in tablet landscape view with only 1 item in cart)

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to your cart
- Navigate to: [Payment step](https://www.merlinspotions.com/checkout/payment/)
- confirm that the items checked off in the WEB-1099 JIRA ticket have been fixed correctly
